modules/system/defaults/NSGlobalDomain.nix

```diff
@@ -257,6 +257,49 @@ in {
       '';
     };
 
+    system.defaults.NSGlobalDomain.AppleInterfaceStyle = mkOption {
+      type = types.nullOr (types.enum [ "Light" "Dark" ]);
+      default = null;
+      description = ''
+        Whether to use Light mode or Dark mode. The default is Light.
+      '';
+    };
+
+    system.defaults.NSGlobalDomain.AppleAccentColor = mkOption {
+      type = types.nullOr (types.enum [ (-1) 0 1 2 3 4 5 6 ]);
+      default = null;
+      example = "";
+      description = ''
+        Which color to use as the system accent color
+        -1 = Graphite
+        0 = Red
+        1 = Orange
+        2 = Yellow
+        3 = Green
+        4 = Blue
+        5 = Purple
+        6 = Pink
+        The default is blue.
+      '';
+    };
+
+    system.defaults.NSGlobalDomain.AppleHighlightColor = mkOption {
+      type = types.nullOr (types.enum [
+        "0.847059 0.847059 0.862745 Graphite"
+        "1.000000 0.733333 0.721569 Red"
+        "1.000000 0.874510 0.701961 Orange"
+        "1.000000 0.937255 0.690196 Yellow"
+        "0.752941 0.964706 0.678431 Green"
+        "0.968627 0.831373 1.000000 Purple"
+        "1.000000 0.749020 0.823529 Pink"
```

Comment:
types.enum means these are the only "valid" options, is that correct?

Reply:
There is also the option to select from a color picker, which sets a value like `"0.549003 0.000000 0.180303 Other"`.